prow/config/config.go

```diff
@@ -2546,6 +2546,13 @@ func ContextDescriptionWithBaseSha(humanReadable, baseSHA string) string {
 	var suffix string
 	if baseSHA != "" {
 		suffix = contextDescriptionBaseSHADelimiter + baseSHA
+		if len(humanReadable+suffix) < 140 {
```

Comment:
140 is a const at https://github.com/kubernetes/test-infra/pull/21704/files#diff-88e6b42a331a7aa9260f9a830961e3ba3c77e0e6d781465378b510d7abc1517cR2574

Reply:
good point, updated. I've also added some godocs to clarify the reason because it might be confusing from just looking at the code.